doxia-modules/doxia-module-xdoc/src/test/java/org/apache/maven/doxia/module/xdoc/XdocSinkTest.java

```diff
@@ -208,6 +208,19 @@ protected String getTableBlock(String cell, String caption) {
                 + "</td></tr></table>";
     }
 
+    @Override
+    protected String getTableWithHeaderBlock(String... rowPrefixes) {
+        return "<table border=\"0\">\n<tr>\n<th>" + rowPrefixes[0] + "0</th>\n<th>" + rowPrefixes[0] + "1</th>\n<th>"
```

Comment:
That border attribute should already be code by now. Is it still emitted? It is deprecated and one should use CSS.

Reply:
This is just what is emitted currently for the newly added test in the `AbstractParserTest`. Fixing the output should be done separately if you consider it wrong.

Reply:
Alright, let's leave as-is.